clients/admin-ui/src/features/common/Icon/HorizontalLine.tsx

```diff
@@ -0,0 +1,26 @@
+import { Icon } from "@fidesui/react";
+
+const HorizontalLine = (props: any) => {
```

Comment:
I have to pass a color prop to the horizontal line to change it when it's the active step, but if you have a better solution, I'm happy to hear!

Reply:
I think createIcon just magically lets you give it a color. For instance, I did that to the QuestionIcon [here](https://github.com/ethyca/fides/blob/8188475d88fdc71981c12573b4b46b71f1cc3905/clients/admin-ui/src/features/common/QuestionTooltip.tsx#L8)

Reply:
Do you know what this would look like with using createIcon? It doesn't appear to take an svg or line attribute, so I can't copy/past the svg information into createIcon with this line.

For example:
```export default createIcon({
  displayName: "HorizontalLineIcon",
  viewBox: "0 0 125 2",
  svg: (
    <svg
      width="125"
      height="2"
      viewBox="0 0 125 2"
      xmlns="http://www.w3.org/2000/svg"
    >
      <line
        x1="0.28418"
        y1="1"
        x2="124.558"
        y2="1"
        stroke={"#CBD5E0"}
        strokeWidth="2"
      />
    </svg>
  ),
});```

I see `path` and `d` are available, but neither of these attributes exist on this svg in particular when copying from figma.

Reply:
hm okay yeah I haven't messed around with it other than just passing it a color prop and watching it magically work before. SVG's handle colors differently, and might just pass it as the stroke prop, but if it doesn't look right, I don't think it should stop this PR going through, but might be something we want to look into in the future